### DISCUSSION TITLE
Show planned revisions in Publishpress Planner

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/PublishPress-Planner#1362] [Feature Requests]
> Hi everyone,
> 
> We use Publishpress Revisions to update regularily our pages and posts, and Publishpress Planner for new contents. Those plugins are really the ones we use almost everyday with our team 😉
> 
> It would be great to have the planned revisions of already published pages shown on the content calendar, isn’t it ?
> 
> Thanks
> Nico

Discussion: https://wordpress.org/support/topic/feature-request-show-planned-revision-in-publishepressplanner/

**Comment by nfavrod** [comment]
Hi ! Thanks @rizaardiyanto1412 for my request relaying here !
Maybe it would be considered in a largest release of planner -> [
Integration with the PublishPress Calendar #98 ](https://github.com/publishpress/PublishPress-Revisions/issues/98 )

**Comment by rizaardiyanto1412** [comment]
For that, let @stevejburge as the product manager decide it.
Thanks
